runtime/hydradx/src/assets.rs

```diff
@@ -985,3 +992,141 @@ impl pallet_xyk::Config for Runtime {
 	type NonDustableWhitelistHandler = Duster;
 	type OracleSource = XYKOracleSourceIdentifier;
 }
+
+parameter_types! {
+	pub const ReferralsPalletId: PalletId = PalletId(*b"referral");
+	pub RegistrationFee: (AssetId,Balance, AccountId)= (NativeAssetId::get(), 222_000_000_000_000, TreasuryAccount::get());
+	pub const MaxCodeLength: u32 = 7;
+	pub const ReferralsOraclePeriod: OraclePeriod = OraclePeriod::Short;
```

Comment:
i think that short is too short, ten minutes or even longer seems to be safer - the oracle period should roughly match how often we will be converting it, and that will be more than every 20 blocks

Reply:
changed to 10mints. We dont have longer one in hydra